tensorboard/webapp/runs/views/runs_table/regex_edit_dialog.ng.html

```diff
@@ -14,23 +14,38 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<h1 mat-dialog-title>Color runs by regex</h1>
+
 <mat-dialog-content>
+  <p>Enter a regex with capturing groups to match against run names:</p>
   <mat-form-field>
     <input
       matInput
       #regexStringInput
-      placeholder="(\w+/).+"
       value="{{ regexString }}"
       (keydown.enter)="onEnter($event.target.value)"
       cdkFocusInitial
     />
   </mat-form-field>
 </mat-dialog-content>
 
-<div mat-dialog-actions>
+<div class="example-details">
+  <p>
+    Each matching run will be assigned a color based on the "key" formed by its
+    matches to the capturing groups. <br />
+    <button mat-stroked-button (click)="fillExample()">
+      Try <code>(train|eval)</code>
+    </button>
+    to assign all runs containing <code>train</code> to one color and all runs
+    containing <code>eval</code> to another color.
```

Comment:
As you said earlier, we may want to have a theme variable for styling the `code` so it looks more like GitHub `code`.

Reply:
defer the changes to next pr